<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IND-CHN-LT10552" ignored="1" name="Android-Web" tests="2" failures="2" timestamp="26 Feb 2021 10:54:28 GMT" time="262.343" errors="0">
  <testcase name="TC_tenantRegistration" time="118.296" classname="TestCases.TestCases">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at utilities.Common.testFailed(Common.java:146)
	at TestCases.TestCases.TestEnd(TestCases.java:114)
	at TestCases.TestCases.TC_tenantRegistration(TestCases.java:74)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- TC_tenantRegistration -->
  <testcase name="TC_tenantLogin" time="112.677" classname="TestCases.TestCases">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at utilities.Common.testFailed(Common.java:146)
	at TestCases.TestCases.TestEnd(TestCases.java:114)
	at TestCases.TestCases.TC_tenantLogin(TestCases.java:91)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- TC_tenantLogin -->
  <testcase name="TC_tenantGroupRequest" time="0.0" classname="TestCases.TestCases">
    <ignored/>
  </testcase> <!-- TC_tenantGroupRequest -->
</testsuite> <!-- Android-Web -->
